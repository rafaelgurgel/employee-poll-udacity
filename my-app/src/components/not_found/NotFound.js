// src/components/NotFound.js
import React from 'react'
import './not_found.css'

export default function NotFound() {
  return (
    <div className="notfound-container">
      <h2 className="notfound-text">404 - Not Found</h2>
    </div>
  )
}
