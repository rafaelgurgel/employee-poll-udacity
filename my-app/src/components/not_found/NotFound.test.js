import React from 'react'
import { render } from '@testing-library/react'
import NotFound from './NotFound'
import '@testing-library/jest-dom'

describe('NotFound Component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<NotFound />)
    expect(asFragment()).toMatchSnapshot()
  })
})
