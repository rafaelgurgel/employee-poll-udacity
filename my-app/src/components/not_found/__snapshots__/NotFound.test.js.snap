// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="notfound-container"
  >
    <h2
      class="notfound-text"
    >
      404 - Not Found
    </h2>
  </div>
</DocumentFragment>
`;
